scripts/updateLicenseHeaders.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import platform
+import subprocess
+import datetime
+from string import Template
+import re
+
+def cmd(args):
+    shell = platform.system() == 'Windows'
+    p = subprocess.Popen(args, shell=shell, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = p.communicate()
+    if p.returncode != 0:
+        raise Exception('cmd(): {0} failed with status {1}: {2}'.format(args, p.returncode, stderr))
+    return stdout.decode('utf-8')
+
+
+licenseHeaderTemplate = Template(
+"""${commentChar} Copyright ${copyright} The Mumble Developers. All rights reserved.
+${commentChar} Use of this source code is governed by a BSD-style license
+${commentChar} that can be found in the LICENSE file at the root of the
+${commentChar} Mumble source tree or at <https://www.mumble.info/LICENSE>.""")
+
+blackListFiles = []
+blackListExtensions = [".md", ".plist", ".o", ".in", ".ui", ".xml", ".ts", ".png", ".jpg", ".ogg", ".mp3",
+        ".ico", ".pyc", ".desktop", ".rtf", ".protocol", ".ini", ".conf", ".service", ".json", ".bmp", ".reg",
+        ".manifest", ".php", ".sqlite", ".sdef", ".icns", ".qrc", ".tmpl", ".oga", ".logrotate", ".svg",
+        ".xpm", ".txt"]
```

Comment:
I would use a whitelist instead, as it's more probable that we add new extensions that are not supposed to be taken into account by the script.

Also, the extensions we want to take into account appear to be more than the one we want to avoid.